docs/src/Tabs.doc.js

```diff
@@ -36,6 +36,11 @@ card(
         description:
           'If your app uses a tool such as react-router to navigate between pages, be sure to use onChange to navigate instead of getting a full page refresh with href',
       },
+      {
+        name: 'icon',
```

Comment:
@danignibus `icon` should be reserved for icons listed in https://gestalt.netlify.app/#/components/Icon

Could we rename this to `indicator: 'dot'`? /cc @ashleyseto